include/zephyr/bluetooth/audio/bap.h

```diff
@@ -50,6 +50,63 @@ extern "C" {
 /** An invalid Broadcast ID */
 #define BT_BAP_INVALID_BROADCAST_ID 0xFFFFFFFFU
 
+/**
+ * @name Suggested connection and advertising intervals suited for coexistence with audio
+ *
+ * Suggest use is to use this as both the minimum and maximum interval for best results.
+ * The intervals are suited for both unicast and broadcast.
+ * @{
+ */
+/**
+ * @brief Fast interval suited for coexistence with 10ms SDU intervals
+ *
+ * This works well with ISO intervals of 10, 20 and 30 milliseconds for broadcast and unicast
```

Comment:
```suggestion
 * This works well with ISO intervals of 10, 20 and 30 milliseconds
```

The intro already says these values are good for both broadcast and unicast